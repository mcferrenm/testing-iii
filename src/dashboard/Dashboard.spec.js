import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from './Dashboard';

describe('matches snapshot', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<Dashboard />);

    expect(tree.toJSON()).toMatchSnapshot();
  })
});