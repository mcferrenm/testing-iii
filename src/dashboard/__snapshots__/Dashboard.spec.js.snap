// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot matches snapshot 1`] = `
Array [
  <div
    className="display panel"
    data-testid="display panel"
  >
    <div
      className="led green-led"
    >
      Unlocked
    </div>
    <div
      className="led green-led"
    >
      Open
    </div>
  </div>,
  <div
    className="controls panel"
    data-testid="controls panel"
  >
    <button
      className="toggle-btn"
      disabled={true}
      onClick={[Function]}
    >
      Lock Gate
    </button>
    <button
      className="toggle-btn"
      disabled={false}
      onClick={[Function]}
    >
      Close Gate
    </button>
  </div>,
]
`;
